lib/db/mysql.js

```diff
@@ -701,6 +701,35 @@ module.exports = function (log, error) {
     return this.write(UPDATE_LOCALE, [data.locale, uid])
   }
 
+  var CREATE_UNBLOCK_CODE = 'CALL createUnblockCode_0(?, ?, ?)'
+
+  MySql.prototype.createUnblockCode = function (uid, code) {
+    return this.write(
+      CREATE_UNBLOCK_CODE,
+      [ uid, code, Date.now() ],
```

Comment:
I've wondered similar, perhaps treating them similar to passwords. It doesn't seem like `tokens` are hashed (in this repo).